recipes/vk-bootstrap/all/conanfile.py

```diff
@@ -88,21 +85,28 @@ def source(self):
     def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["VK_BOOTSTRAP_TEST"] = False
-        vulkan_headers = self.dependencies["vulkan-headers"]
-        includedirs = ";".join(
-            [os.path.join(vulkan_headers.package_folder, includedir).replace("\\", "/")
-             for includedir in vulkan_headers.cpp_info.includedirs],
-        )
-        if Version(self.version) < "0.3.0":
-            tc.variables["Vulkan_INCLUDE_DIR"] = includedirs
-        else:
+        tc.variables["VK_BOOTSTRAP_WERROR"] = False
+        if Version(self.version) < Version("1.0"):
+            vulkan_headers = self.dependencies["vulkan-headers"]
+            includedirs = ";".join(
+                [os.path.join(vulkan_headers.package_folder, includedir).replace("\\", "/")
+                for includedir in vulkan_headers.cpp_info.includedirs],
+            )
             tc.variables["VK_BOOTSTRAP_VULKAN_HEADER_DIR"] = includedirs
-        if Version(self.version) >= "0.4.0":
-            tc.variables["VK_BOOTSTRAP_WERROR"] = False
         tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def _source_patches(self):
+        if Version(self.version) >= "v1.3.266":
```

Comment:
```suggestion
        if Version(self.version) >= "1.3.266":
```
The version has bad formatting